### DISCUSSION TITLE
Community chat

### DISCUSSION THREAD

**Discussion by olorin37** [stedolan/jq#2921] [Ideas]
Is there any plan to create matrix channal or discord server for the jq community. Some ideas could be polished there.

Of course this place is also feasible.

**Comment by wader** [comment]
Hey, there is a discord server already, see https://github.com/jqlang/jq#community--support

**Comment by emanuele6** [comment]
There has also been for many years an official `#jq` IRC channel on [libera.chat](https://libera.chat). (previously on freenode before 2021).
